src/__utils__/helpers.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * Generates a random token with a prefix to support
+ * unique, idempotent local testing.
+ * @param prefix 
+ * @returns 
+ */
+export function generateTestToken(prefix: string): string {
+  return `${prefix}-${Date.now().toString()}`
```

Comment:
Can this is called at the exact moment from two tests? If so, might be worth adding a random number as well

Reply:
I think the prefix should suffice to ensure that a unique token is being generated for the exact timestamp the test starts, so long as the usage of this method passes in a unique token per test case. Happy to add a random number as well, but I believe it should be distinct enough